### PR TITLE
Handle RSS Feeds in daemon mode

### PR DESCRIPTION
**Pull request by imattau**
A2PS can handle RSS feed url's e.g example.com/feed or example.com/rss.

Implements daemonize to run in the background.

### CODE REVIEW

**Comment by edhelas** [comment]
Sorry, totally missed that one, I'll try to review it asap :)

File: atomtopubsub.py [hunk 1]
```diff
@@ -25,29 +49,52 @@ def setup_logging(level):
...
+        
+        
         if f.bozo == 1:
             print('XML Error')
             if hasattr(f.bozo_exception, 'getMessage'):
                 print(f.bozo_exception.getMessage())
             if hasattr(f.bozo_exception, 'getLineNumber'):
                 print('at line %s' % f.bozo_exception.getLineNumber())
-
+
```
Comment (edhelas) [inline atomtopubsub.py:74]: Can you please ensure that you're not adding spaces where they are not needed :) ?

File: atomtopubsub.py [hunk 2]
```diff
@@ -115,6 +186,10 @@ def main():
...
     xmpp.process()
 
+curr_dir = getcwd() #Needed so that A2SP can find the config files etc when using daemonize
+
+daemon = Daemonize(app='atomtopubsub', pid = pid, action = main, foreground=True, chdir= curr_dir)
+daemon.start()
 
-if __name__ == '__main__':
-    main()
+#if __name__ == '__main__':
```
Comment (edhelas) [inline atomtopubsub.py:194]: Commented code ?

**Comment by sjehuda** [comment]
Did you also take care of [this](https://github.com/edhelas/atomtopubsub/issues/7)? 

For that task, I've made [this check](https://github.com/sjehuda/slixmpp-news-bot/blob/main/monitor_feed_updates.py#L155)

**Pull request closed without merging**